src/fhirtypes/StructureDefinition.ts

```diff
@@ -603,6 +603,26 @@ export class StructureDefinition {
     fisher: Fishable
   ): ElementDefinition {
     let matchingSlice = elements.find(e => e.sliceName === pathPart.brackets.join('/'));
+    if (!matchingSlice && pathPart.brackets?.length === 1) {
+      // If the current element is a child of a slice, the match may exist on the original
+      // sliced element, search for that here
+      for (const e of elements) {
+        const connectedSliceElement = e.findConnectedSliceElement();
+        const matchingConnectedSlice = connectedSliceElement
```

Comment:
This `if` block has the same condition as the next `if` block, so I think they can be combined.